types/intl-tel-input/global.d.ts

```diff
@@ -0,0 +1,395 @@
+declare namespace IntlTelInput {
```

Comment:
I think this structure leaves IntlTelInput global even when the consumer uses modules otherwise. You can test this with a test file that imports something else, but not IntlTelInput. I suspect it will still be accessible.

The fix is to put the namespace in the same file instead of triple slash referencing it, I think.